docs/source/track_errors/divisions.rst

```diff
@@ -0,0 +1,95 @@
+Division Errors
+===============
+Note: These are annotated on the parent nodes.
+
+True Positive
+-------------
+
+A true positive division is a division event in which the parent and both daughters match between the ground truth and predicted graphs. True positive divisions are annotated on both the ground truth and predicted graphs.
```

Comment:
Can you say that they are annotated on the parent node specifically? That's what we do, right?